### ISSUE TITLE
[Feature] Color painting text

### ISSUE THREAD

**Issue by candre23** [bambulab/BambuStudio#621]
Add the ability to paint text onto a model.  Seems simple and obvious, doesn't it?

**Comment by YifanWuBambu** [comment]
Currently, Bambu Studio supports add text and quickly specify its color by numeric key #1~9.
Maybe you could use this feature first.

I have added text painting (texturing the text to a model) to our plan list.

**Comment by candre23** [comment]
Yes, I am aware that text models can be added with the text tool and merged with other models to create embossed text.  But I am referring to simply "painting" text on to a model by changing the filament used for the region covered by the text, without adding or subtracting material.  

The circle paint tool is able to change the color/material for a surface region without modifying its geometry.  I am proposing a text paint tool which would do the exact same thing - except instead of the brush being a circle, it would be text of the user's choosing.

**Comment by SaltWei** [comment]
![image](https://github.com/bambulab/BambuStudio/assets/110660407/255fde64-fa34-440c-82cc-38232ff7f320)
You can set the text to be modifier so that the text surface can be aligned with model surface absolutely.
![image](https://github.com/bambulab/BambuStudio/assets/110660407/fc334d9d-9fbb-4001-b968-2b2012d52ace)

**Comment by SaltWei** [comment]
> ![image](https://user-images.githubusercontent.com/110660407/273423577-255fde64-fa34-440c-82cc-38232ff7f320.png) You can set the text to be modifier so that the text surface can be aligned with model surface absolutely, just like painting ![image](https://user-images.githubusercontent.com/110660407/273423609-fc334d9d-9fbb-4001-b968-2b2012d52ace.png)
